### PR TITLE
support ListOptions for ListMembershipsForBillableGroupMember

### PR DESCRIPTION
**Pull request by Iridias**
Just realized, that there was no pagination-support for this function/endpoint.
Forgot to add it in my initial commit - my bad :-/

### CODE REVIEW

File: group_members.go [hunk 1]
```diff
@@ -258,14 +258,14 @@ func (s *GroupsService) ListBillableGroupMembers(gid interface{}, opt *ListBilla
 //
 // GitLab API docs:
 // https://docs.gitlab.com/ee/api/members.html#list-memberships-for-a-billable-member-of-a-group
-func (s *GroupsService) ListMembershipsForBillableGroupMember(gid interface{}, user int, options ...RequestOptionFunc) ([]*BillableUserMembership, *Response, error) {
+func (s *GroupsService) ListMembershipsForBillableGroupMember(gid interface{}, user int, opt *ListOptions, options ...RequestOptionFunc) ([]*BillableUserMembership, *Response, error) {
```
Comment (svanharmelen) [inline group_members.go:261]: Please check the code for examples, but you should not use the `ListOptions` struct directly. Instead you should define a `ListMembershipsForBillableGroupMemberOptions` struct which could be just `type ListMembershipsForBillableGroupMemberOptions = ListOptions`...

Comment (Iridias) [reply group_members.go:261]: Ah, ok - wasn't sure about that, as I found some other occurrences, where it's used directly (e.g. in `ListProjectStatusChecks`) 
I'll change it as suggested.

Comment (svanharmelen) [reply group_members.go]: As far as I can see there are 3 other location where it slipped in, but good to not let the number increase 😏

**Review by svanharmelen — APPROVED** [review]
Thanks 👍🏻

**Pull request merged**